app/javascript/orangelight/availability.es6

```diff
@@ -35,25 +35,24 @@ export default class AvailabilityUpdater {
     // a search results page or a call number browse page
     if ($(".documents-list").length > 0) {
       const bib_ids = this.record_ids();
-      if (bib_ids.length < 1) { return; }
-      url = `${this.bibdata_base_url}/bibliographic/availability.json?bib_ids=${bib_ids.join()}`;
-      return $.getJSON(url, this.process_results_list)
-        .fail((jqXHR, textStatus, errorThrown) => {
-          if (jqXHR.status == 429) {
-            if (allowRetry) {
-              console.log(`Retrying availability for records ${bib_ids.join()}`);
-              window.setTimeout(() => {
-                this.update_availability_retrying();
-                this.request_availability(false);
-              }, 1500);
-            } else {
-              console.error(`Failed to retrieve availability data for bibs (retry). Records ${bib_ids.join()}: ${errorThrown}`);
-              this.update_availability_undetermined();
-            }
-            return;
-          }
-          return console.error(`Failed to retrieve availability data for bibs. Records ${bib_ids.join(", ")}: ${errorThrown}`);
-        });
+      if (bib_ids.length < 1) {
+        return;
+      }
+
+      const batch_size = 10;
+      const batches = this.ids_to_batches(bib_ids, batch_size);
+      console.log(`Requested at ${new Date().toISOString()}, batch size: ${batch_size}, batches: ${batches.length}, ids: ${bib_ids.length}`);
```

Comment:
I left a few `console.log()` in place on purpose to be able to validate the logic in production for a few days after we deploy this to production. We should be able to remove them later on.